src/tools/clippy/Cargo.toml

```diff
@@ -22,31 +22,31 @@ path = "src/driver.rs"
 
 [dependencies]
 # begin automatic update
-clippy_lints = { version = "0.1.50", path = "clippy_lints" }
+clippy_lints = { version = "0.1", path = "clippy_lints" }
 # end automatic update
```

Comment:
Can you remove those comments, while you're at it. This is definitely not automatically updated :smile: